docs/data_format_changes/i1620-remove-first-crdt-byte.md

```diff
@@ -0,0 +1,3 @@
+# Remove the first CRDT byte from field encoded values
+
+The first CRDT byte was legacy code and no longer necessary as we have this information independently available via the client.FieldDescription, since the FieldDescription.Typ is the exact same value.
```

Comment:
thought: For the future (I wouldn't bother changing this now as atm it doesnt really matter), but these docs are really for our users, and I think this is worded a little too internally and might not be very helpful for external persons.

Reply:
I always saw this as internal dev information. It's an interesting point of view :)

Reply:
Is kind of like a change-log, of really really important changes.  Users will likely want to know when, how and maybe why their persisted data has been broken between versions, regardless as to whether or not we have any migrations to help them migrate.

Reply:
We'll have to surface these elsewhere because I'm certain that users won't come looking for those in a repo sub folder.